DDS-1430 [Tabs] [Tab Panels] [Tab] Retouching

poetrainy opened the pull request:
## Related Ticket(s) and Links

- JIRA Ticket: [DDS-1430](https://daikinsvoil.atlassian.net/browse/DDS-1430)
- Figma design: [Figma](https://www.figma.com/design/VyaaU8Ta9yzyf0PsURWSSf/branch/tYQLiCR0vIE6xNh4EJoHxq/DDS%3A-Design-Kit?node-id=31547-29653&node-type=section&t=2ZL31Q0XlwQgTrqB-0)

## Checklist

- [x] I have added a changeset file that describes the changes.
- [x] I have performed a self-review of my code.
- [x] I have added/updated appropriate regression tests.
- [x] I have confirmed that there are no accessibility warnings that can be addressed.
- [x] I have confirmed that the bug fix does not affect the keyboard accessibility of the component.
- [x] I have confirmed that the fix does not introduce any new visual regressions.

[DDS-1430]: https://daikinsvoil.atlassian.net/browse/DDS-1430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

yodas7 commented on src/components/tab/daikin-tab.ts:
```diff
@@ -73,27 +81,16 @@ export class DaikinTab extends LitElement {
-
-    :host([size="condensed"]) {
-      height: 40px;
+      width: max-content;
```
Maybe we don't need this?

yodas7 commented on src/components/tabs/daikin-tabs.ts:
```diff
@@ -19,35 +19,35 @@ import { scrollIntoViewOnlyParent } from "./scroller";
  *
  * @slot - A slot for tab buttons. Place `daikin-tab` elements here.
- * @slot panels - A slot for a panel switcher component. Place `daikin-panel-switcher` element(s) here.
+ * @slot panels - A slot for a panel switcher component. Place `daikin-tab-panels` element(s) here.
```
```suggestion
 * @slot panels - A slot for tab panels component. Place a `daikin-tab-panels` element here.
```

yodas7 commented on src/components/tab/daikin-tab.ts:
```diff
@@ -10,50 +10,58 @@ const cvaTab = cva(
+    "focus-visible:before:w-full",
+    "focus-visible:before:h-[calc(100%_-_4px)]",
+    "focus-visible:before:border",
+    "focus-visible:before:border-solid",
```
nit.

```suggestion
```

yodas7 commented on .changeset/olive-countries-worry.md:
```diff
@@ -0,0 +1,8 @@
+"@daikin-oss/design-system-web-components": minor
+---
+
+**BREAKING CHANGE** Tabs: The name of the component has been changed from `Tab Group` to `Tabs`. (DDS-1450)
```
```suggestion
**BREAKING CHANGE** Tabs: The name of the component has been changed from `Tab Group` to `Tabs`. Use `<daikin-tabs>` instead of `<daikin-tab-group>`. (DDS-1450)
```

yodas7 commented on .changeset/olive-countries-worry.md:
```diff
@@ -0,0 +1,8 @@
+---
+
+**BREAKING CHANGE** Tabs: The name of the component has been changed from `Tab Group` to `Tabs`. (DDS-1450)
+**BREAKING CHANGE** Tab Panels: The name of the component has been changed from `Panel Switcher` to `Tab panels`. (DDS-1450)
```
```suggestion
**BREAKING CHANGE** Tab Panels: The name of the component has been changed from `Panel Switcher` to `Tab panels`. Use `<daikin-tab-panels>` instead of `<daikin-panel-switcher>`. (DDS-1450)
```

yodas7 reviewed (commented):
I feel like the `change` event should be a plain `new Event("change")` like the other components.
I initially designed it with the [`toggle`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/toggle_event) / [`beforetoggle`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/beforetoggle_event) events in mind, but I thought it would be strange to have the same event name as the others but a different interface.

I don't know what to do about the `beforechange` event. I don't think we need both `newValue` and `oldValue`, because we can get `oldValue` from `event.target.value`. I think that `newTab: DaikinTabElement` is more informative and better than `newValue`.

poetrainy commented:
@yodas7 
I deleted `oldValue`. I also added newTab, but because there were places where I couldn't get `event`, I allowed for undefined and left `newValue`.

yodas7 commented:
@poetrainy 
There seems to be a misunderstanding. The word `event` was meant to refer to the recipient's variable. Since it is probably difficult to convey, I will make the change directly.

yodas7 reviewed (approved):
LGTM
